src/status_im/wallet/core.cljs

```diff
@@ -1075,32 +1075,14 @@
 (re-frame/reg-fx
  ;;TODO: this could be replaced by a single API call on status-go side
  :wallet-legacy/initialize-wallet
- (fn [[network-id network callback]]
+ (fn [[network-id _network callback]]
```

Comment:
:+1:

Reply:
Maybe we can remove `_network` param if we are not using it?

Reply:
Good shout!
I was intending to make another issue to do a further clean up of those initialisation events in. I wanted to get this bug removed asap because it's impacting many current issues

Reply:
I will start cleaning up (removing) the old wallet code and create initialize events with RPC calls needed for the new wallet.

It's tracked here: https://github.com/status-im/status-mobile/issues/17985

Reply:
Awesome, thanks @smohamedjavid !